SPECS/expat/expat.spec

```diff
@@ -53,6 +53,7 @@ rm -rf %{buildroot}/%{_docdir}/%{name}
 %defattr(-,root,root)
 %doc AUTHORS Changes
 %{_bindir}/*
+%exclude %{_mandir}/man1/xmlwf.1.gz
```

Comment:
Could you remove this file in the `%install` step instead? For the sake of getting rid of the files we don't need consistently in one place (we already remove `*.la` and `%{buildroot}/%{_docdir}/%{name}` from there).

Reply:
For some reason, the build fails when removing the file from the `%install` step and I am not sure why, any ideas?

Reply:
Resolving this since it turns out that we do need to include this file